daemon/execdriver/lxc/driver.go

```diff
@@ -773,15 +787,17 @@ func (d *driver) generateEnvConfig(c *execdriver.Command) error {
 }
 
 // Clean not implemented for lxc
-func (d *driver) Clean(id string) error {
+func (d *Driver) Clean(id string) error {
 	return nil
 }
 
+// TtyConsole is for when using a container interactively
```

Comment:
I think we should use master and slave instead of read and write, it's more standard way to describe pty, isn't it?